pandas/core/arrays/integer.py

```diff
@@ -280,7 +280,7 @@ def _coerce_to_ndarray(self):
         data[self._mask] = self._na_value
         return data
 
-    __array_priority__ = 1  # higher than ndarray so ops dispatch to us
+    __array_priority__ = 1000  # higher than ndarray so ops dispatch to us
```

Comment:
should we just put this in the base class? (for the ops mixin)

Reply:
That seems a little too invasive for a base class. I’d rather leave that up to the subclasser.

Reply:
so what arithmetic subclass would not want this set?

is there an example?

Reply:
I just find this a detail which would likely be forgotten in any subclass, I don't see a harm and much upset in setting it onthe base class (you can always unset if you really really think you need to).

Reply:
Can you unset it?

Reply:
I don't really know if setting `__array_priority__ = 0` is enough to "unset" it, and I don't know what all setting `__array_priority__` in the first place opts you into.

Reply:
can you document this in the Mixin itself though (if you are not going to set it by defaulrt). It is so non-obvious that you need to do this.